systest/backup/advanced-scenarios/127-Namespace/backup_test.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	restoreLocation = "/data/backup/"
+	backupDst       = "/data/backup/"
+	headerAlpha1    = http.Header{}
+	headerAlpha2    = http.Header{}
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+	ShellToUse      = "bash"
+)
+
+func TestDeletedNamespaceID(t *testing.T) {
+	err := common.RemoveContentsOfPerticularDir(t, "./data/backup/")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	// *****************Get JWT-Token
```

Comment:
I find these asterisk-based lines distracting

Reply:
**Resolved:**  Removed all distracting comments from the code.